iot_packet_verifier/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+ [package]
+name = "iot-packet-verifier"
+version = "0.1.0"
+edition = "2021"
+description = "Packet verification for IOT"
+license = "Apache-2.0"
+
+[dependencies]
+anyhow = {workspace = true}
+anchor-lang = {workspace = true}
+anchor-client = "0.26"
+async-trait = {workspace = true}
+clap = {workspace = true}
+config = {workspace = true}
+chrono = {workspace = true}
+db-store = {path = "../db_store"}
+futures = {workspace = true}
+futures-util = {workspace = true}
+file-store = {path = "../file_store"}
+helium-proto = {workspace = true}
+helium-crypto = {workspace = true, features = ["sqlx-postgres", "multisig", "solana"]}
+solana-client = {workspace = true}
+solana-sdk = {workspace = true}
+solana-program = "1.11"
+spl-token = "3.5.0"
+node-follower = {path = "../node_follower"}
+poc-metrics = {path = "../metrics"}
+prost = {workspace = true}
+serde = {workspace = true}
+sqlx = {workspace = true}
+thiserror = {workspace = true}
+tokio = {workspace = true}
+tonic = {workspace = true}
+tracing = {workspace = true}
+tracing-subscriber = {workspace = true}
+triggered = {workspace = true}
+sha256 = "1.1.1"
+
+data-credits = {git = "https://github.com/helium/helium-program-library.git"}
+helium-sub-daos = {git = "https://github.com/helium/helium-program-library.git"}
```

Comment:
how does this one work with them being the same repo?

Reply:
When the git repo is a workspace, it will use the package name to find the source